node-repository/src/main/java/com/yahoo/vespa/hosted/provision/lb/SharedLoadBalancerService.java

```diff
@@ -28,12 +29,14 @@ public SharedLoadBalancerService(String vipHostname) {
 
     @Override
     public LoadBalancerInstance create(LoadBalancerSpec spec, boolean force) {
+        if (spec.settings() != LoadBalancerSettings.empty) throw new IllegalArgumentException("custom load balancer settings are not supported with " + getClass());
```

Comment:
Should be `equals`, so non-empty settings always throws.

Reply:
Yes, replaced with `isEmpty()` method.